pom.xml

```diff
@@ -82,7 +82,7 @@
         </version.org.jboss.logging.jboss-logging-annotations>
         <version.org.jboss.logging.jboss-logging-tools>2.2.1.Final</version.org.jboss.logging.jboss-logging-tools>
         <version.org.jboss.logging.jboss-logging>3.5.3.Final</version.org.jboss.logging.jboss-logging>
-        <version.org.jboss.resteasy>6.2.7.Final</version.org.jboss.resteasy>
+        <version.org.jboss.resteasy>7.0.0.Alpha1-SNAPSHOT</version.org.jboss.resteasy>
```

Comment:
Does it not work with 6.2.7.Final? I'm not sure when we'll have a 7.0.0 out.

Reply:
It won't work with `6.2.7.Final` because of the dependency conflict:

```
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.119 s <<< FAILURE! -- in org.jboss.resteasy.test.spring.deployment.ContextRefreshDependenciesInDeploymentTest
[ERROR] org.jboss.resteasy.test.spring.deployment.ContextRefreshDependenciesInDeploymentTest.testContextRefresh -- Time elapsed: 0.052 s <<< ERROR!
java.lang.IllegalStateException: Multiple TestRunners found, only one allowed. Check your classpath
```